Testscripts/Linux/ETHTOOL-GET-STATISTICS.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License.
+#############################################################################
+ChangeMTU(){
+    test_iface=$1
+    value=$2
+LogMsg "Setting MTU $value on $test_iface" >> ~/summary.log
+sudo ip link set dev $test_iface mtu $value
+if [ $? -ne 0 ]; then
+    LogErr "Failed to change MTU !" >> ~/summary.log
+    SetTestStateFailed
+    exit 0
+fi
+}
+
+# Source utils.sh
+. utils.sh || {
+    echo "Error: unable to source utils.sh!"
+    echo "TestAborted" > state.txt
+    exit 0
+}
+
+# Source constants file and initialize most common variables
+UtilsInit
+
+# Check if ethtool exist and install it if not
+VerifyIsEthtool
+
+net_interface=eth0
+# Verify the new NIC received an IP v4 address
+LogMsg "Verify the new NIC has an IPv4 address"
+ip addr show ${net_interface} | grep "inet\b" > /dev/null
```

Comment:
these 4 lines are not needed, automation will not get here if interface is not up / does not have IP.
remove lines 31-35.

Reply:
fixd